types/Cargo.toml

```diff
@@ -13,6 +13,7 @@ resolver = "2"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+askama         = "0.10.5"
```

Comment:
that should be an optional dependency, and should be placed at the bottom of the `[dependencies]` list.

Reply:
Done.